spec/index.spec.js

```diff
@@ -0,0 +1,77 @@
+describe("AcnCheckController", function () {
+  beforeEach(module("ACNApp"));
+
+  var $rootScope, $controller, $scope, controller;
+
+  beforeEach(inject(function (_$controller_, _$rootScope_) {
+    $rootScope = _$rootScope_;
+    $controller = _$controller_;
+    $scope = $rootScope.$new();
+    controller = $controller("mainController", {
+      $scope: $scope,
+    });
+  }));
+
+  describe("$scope.onChangeAcn", function () {
```

Comment:
usually we will test the service for this business logic because the service could be used on other controller.
and add another test for controller to test different logic for the controller itself. This PR is looking really good. This sandbox project main goal is to learn how to use our dev tools. Probably left this open and let other team to give feedback and apply them on your telflow portal PR 👍